chrome/content/qubesattachment.js

```diff
@@ -32,7 +32,7 @@ var qubesattachment = {
 
               var process = Components.classes["@mozilla.org/process/util;1"].createInstance(Components.interfaces.nsIProcess);
               var procPath = Components.classes["@mozilla.org/file/local;1"]
-                  .createInstance(Components.interfaces.nsILocalFile);
+                  .createInstance(Components.interfaces.nsIFile);
```

Comment:
Does it work also on older thunderbird versions (as you noted in manifest: >=14)?

Reply:
It appears to work with Thunderbird 14.0, so I am assuming that all intermediate versions would work too.

More details: I created a temporary appVM, and I downloaded Thunderbird 14.0 from [here](https://ftp.mozilla.org/pub/thunderbird/releases/14.0/linux-x86_64/en-US/). Afterwards, I transferred to the appVM an e-mail file (`*.eml`) containing an attachment. I tested the open-attachment-in-disposable-VM feature, and it worked.

Here is a screenshot of the add-on's preferences window along with Thunderbird's version window. Perhaps the screenshot is not sufficient on its own to demonstrate that the changes work though, given that I did not up-version the add-on.

![screenshot](https://user-images.githubusercontent.com/16512453/43872364-6cdcb05e-9b4f-11e8-8f35-b663b233962e.png)